src/test/lib/operators/aggregate_test.cpp

```diff
@@ -25,6 +25,14 @@
 
 namespace hyrise {
 
+class OperatorsAggregateHashTest : public BaseTest {};
```

Comment:
Do you mean `OperatorsAggregateTest`? Because that one is not specific to hashing, but instantiated for both sorting and hashing (which is why I didn't want to add the new test to that test suite)

Reply:
I did not realize that the rest is typed for both operators, so just ignore it